tests/ray_cluster_test.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import print_function
+
+from absl import app
+from absl import flags
+
+from ray.tests.cluster_utils import Cluster
```

Comment:
ERDOS abstractions should not leak anything about the underlying execution system, except when `graph.execute()` is called. 

We should also not depend on test utils, since it is not a stable and documented API.

Reply:
I'm not really sure how else to quickly simulate a Ray cluster without manually SSHing into several different nodes. The Ray cluster abstraction is reliable and widely used across Ray tests. Do you have any suggestions on what could be better?

Anyway, doesn't the `--ray_redis_address` flag already leak information about the underlying execution system?